client/src/components/SearchResults/SearchResults.js

```diff
@@ -0,0 +1,56 @@
+import './SearchResults.css';
+import React from 'react';
+
+// const courses = [
+//   {
+//     "id": 1,
+//     "title": "Algebra I",
+//     "headline":"The entry course to learn algebra",
+//     "url": "https://www.udemy.com/course/algebra-i-beginning-algebra/",
+//     "image_240x135":"https://i.udemycdn.com/course/240x135/34267_f952_15.jpg",
+//     "num_lectures":14,
+//     "num_subscribers":1566
+//   },
+//   {
+//       "id": 2,
+//       "title": "How To Make Money Online In 2020 Without Investment",
+//       "headline":"Easy ways to Generate a Steady Income from Trusted Websites online",
+//       "url": "https://i.udemycdn.com/course/240x135/1687236_bab3.jpg",
+//       "image_240x135":"https://i.udemycdn.com/course/240x135/1687236_bab3.jpg",
+//       "num_lectures":60,
+//       "num_subscribers":1354
+//     }
+//   ];
+
+const SearchResults = ({courses}) => {
+  if (!courses || courses.length === 0) {
+    return (
+    <div className='no-results-container'>
+      <br></br><br></br>
+      <h3>Sorry .. No results found.</h3>
+    </div>
+    );
+  }
+
+  const results = courses.map((course) => (
```

Comment:
wouldn't this just throw an error ? since courses does not exist

Reply:
Yeah I intended to send the dummy courses as a prop from the app.js , since we need that data in another component as well